src/ProgressOnderwijsUtils/Html/HtmlFragmentSanitizeExtension.cs

```diff
@@ -50,12 +50,13 @@ public TagSafety AllowTag(IHtmlElement elem)
                 => TagSafety.SafeToKeep;
 
             public bool AllowAttribute(HtmlAttribute attr)
-                => !attr.Name.EqualsOrdinalCaseInsensitive("style") || SafeStyleRegex.IsMatch(attr.Value);
+                => !attr.Name.EqualsOrdinalCaseInsensitive("style") || attr.Value.Split(new[]{';'}, StringSplitOptions.RemoveEmptyEntries).All(o => SafeStyleRegex.IsMatch(o));
 
             static readonly Regex
                 SafeStyleRegex = new Regex(@"^
-                    \s*margin(-(left|right|top|bottom))?\s*:\s*
-                    \d+(px|em|cm|mm|)\s*;?\s*
+                    (\s*margin(-(left|right|top|bottom))?\s*:\s*\d+(px|em|cm|mm|)\s*)
+                    | (\s*color\s*:\s*[^;]+)

```

Comment:
niet dat ik *denk* dat er een browser is die er iets mee zou doen, maar wat als iemand `color: url(javascript:evilhaxxor())` doet?

Reply:
Als we willekeurige input toestaan heeft dat een commentaar nodig van waarom dat hier veilig is (want in zijn algemeenheid geldt dat niet).